packages/amplify_auth_cognito/android/src/main/kotlin/com/amazonaws/amplify/amplify_auth_cognito/types/FlutterSignInRequest.kt

```diff
@@ -38,5 +40,17 @@ data class FlutterSignInRequest(val map: HashMap<String, *>) {
         }
       }
     }
+    fun checkUser() {
+      try {
+        var user: AuthUser? = Amplify.Auth.currentUser;
+        if (user != null) {
+          throw FlutterInvalidStateException("There is already a user  signed in.", "Sign out before calling sign in.")
```

Comment:
Nit: Is this the same language as iOS?
Shouldn't the recovery suggestion refer to the API name `signIn`:  
```
"Sign out before calling signIn()"
```

Reply:
@Ashish-Nanda It is very close - it corrects a very small grammatical error.  And ios does not use "()".

Reply:
And I think "()" is implied by "calling".